docs/merkletree/treetex/main.go

```diff
@@ -382,5 +382,5 @@ func main() {
 	// arbitrary stream.
 	fmt.Print(preamble)
 	renderTree("", *treeSize, 0)
-	fmt.Println(postfix)
+	fmt.Print(postfix)
```

Comment:
What's the nit here?  Won't this change the output?

Reply:
The extra newline is already in the string literal.